src/Components/Chakra__Select.res

```diff
@@ -628,6 +630,9 @@ let makeProps = (
     /**
       Component Props {Select}
      **/
+    ~errorBorderColor=?errorBorderColor->BorderColor.fromOption,
+    ~focusBorderColor=?focusBorderColor->BorderColor.fromOption,
+    ~iconColor=?iconColor->Color.fromOption,
     ~variant=?variant->Select.Variant.fromOption,
```

Comment:
Not sure if there should be `IconColor`?

Reply:
No problem, this is okay.